src/global/reco/reco.cc

```diff
@@ -3,14 +3,15 @@
 //
 //
 
+
 #include <JANA/JApplication.h>
 #include <edm4eic/Cluster.h>
 #include <edm4eic/EDM4eicVersion.h>
 #include <edm4eic/MCRecoClusterParticleAssociation.h>
 #include <edm4eic/MCRecoParticleAssociation.h>
 #include <edm4eic/ReconstructedParticle.h>
 #include <edm4hep/MCParticle.h>
-#include <algorithm>
+//#include <algorithm>
```

Comment:
Is there a reason to comment this out rather than just delete this include?

Reply:
```suggestion
```